apps/meteor/tests/e2e/retention-policy.spec.ts

```diff
@@ -171,6 +174,15 @@ test.describe.serial('retention-policy', () => {
 				await expect(poHomeChannel.tabs.room.getMaxAgeLabel('15')).toBeVisible();
 				await expect(poHomeChannel.tabs.room.inputRetentionMaxAge).toHaveValue('365');
 			});
+
+			test('should ignore threads be checked accordingly with the global default value', async () => {
```

Comment:
Can we add a case overriding the value and ensuring the value is being overwritten?